.travis.yml

```diff
@@ -11,18 +11,19 @@ notifications:
       - travis-build@platform-notifications.robomaker.aws.a2z.com
 env:
   global:
-    - UPSTREAM_WORKSPACE=file
-    - ROSINSTALL_FILENAME=.rosinstall.master
+    - GITHUB_ACCOUNT=aws-robotics
+  # uncomment PACKAGE_NAME to build unit tests - REQUIRED if you'd like to run unit tests using colcon
```

Comment:
Missing `PACKAGE_NAME` line or `NO_TESTS`, it should have one or the other.

Reply:
Abby explained how it works. `NO_TESTS` skips all tests, passing `PACKAGE_NAME` builds the package with cmake-target of tests (see https://github.com/aws-robotics/travis-scripts/blob/master/ros1_build.sh#L35), so when the package has tests but doesn't need to be built explicitly with them it's ok to omit both.

Reply:
As discussed offline. This package doesn't need `PACKAGE_NAME` or `NO_TESTS`.